torch/testing/_internal/logging_utils.py

```diff
@@ -211,3 +211,33 @@ def ctx_manager():
         return exit_stack
 
     return log_stream, ctx_manager
+
+
+def multiple_logs_to_string(module, *log_options):
```

Comment:
i know this is test code so matters a bit less, but nonetheless: `module: str, *log_options: str) -> Tuple[List[io.StringIO], typing.ContextManager[None]]`